app/views/driver_sessions/sign_up.html.erb

```diff
@@ -0,0 +1,80 @@
+<h1>Sign up</h1>
+
+<form action="/post_driver" method="post">
+
+  <div>
+    <label for="email">Email</label>
+    <input id="email" name="email" placeholder="Choose a email..." type="text">
+  </div>
+
+      
+      
+         
+  
+  <label for="username">
```

Comment:
The indentation on all `<label>`s looks a bit off to me (too small)?

Reply:
You're right! They aren't in the same `div` as the inputs.